tighten MUST wording for network re-attach/PATH_CHALLENGE (#20)
danwing/cidfi: draft-wing-cidfi.md

# Topology Change {#topology}

When the topology changes the client will transmit from a new IP
address -- such as switching to a backup WAN connection, or such as
switching from Wi-Fi to 5G.  If using QUIC, the server will consider

> this a connection migration {{Section 9 of QUIC}} and will issue a

Reviewer: ```suggestion
this as a connection migration ({{Section 9 of QUIC}}) and will issue a
```

PATH_CHALLENGE.  If the client is aware of the topology change (such
as attaching to a different network), the client would also change its
QUIC Destination CID ({{Section 9 of QUIC}}).

> When the CIDFI-aware client determines it has connected to a new

Reviewer: ```suggestion
When the CIDFI-aware client determines that it is connected to a new
```

> network or has received a QUIC PATH_CHALLENGE the CIDFI-aware client

Reviewer: ```suggestion
network or has received a QUIC PATH_CHALLENGE, the CIDFI-aware client
```

> MUST re-discover its CNEs {{discovery}} and continue with normal CIDFI

Reviewer: ```suggestion
MUST re-discover its CNEs ({{discovery}}) and continue with normal CIDFI
```

> processing with those CNEs.

Reviewer: ```suggestion
processing with any discovered CNEs.
```
